src/index.html

```diff
@@ -11,7 +11,159 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="field">
+          How old are You?
+          <input type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date-of-birth"
+          >
+        </label>
+
+        <label class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accepted-term"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <label class="field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required minlength="8"
+            maxlength="30"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <label class="field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="Yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="No"
+          >
+          No
+        </label>
+
+        <label class="field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="favorite-color"
+          >
+        </label>
+
+        <label class="field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="go-to-bed-time"
```

Comment:
You didn't fix all the points from a checklist

Reply:
Forgot  about not using '-', my bad